contrib/mypy/src/python/pants/contrib/mypy/tasks/mypy_task.py

```diff
@@ -194,7 +194,7 @@ def _get_mypy_pex(self, py3_interpreter: PythonInterpreter, *extra_pexes: PEX) -
 
             site.getsitepackages = lambda: sys.path[:]
 
-            
+
```

Comment:
Usually we try to avoid making whitespace changes in unrelated files, even if your editor is doing the right thing here by removing trailing whitespace. This can be undone with `git checkout master -- contrib/mypy/src/python/pants/contrib/mypy/tasks/mypy_task.py`.

Reply:
Thx! Will make note of this going forward